### PR TITLE
{dev} [intel/2016b] CMake/3.6.1 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: easybuild/easyconfigs/c/CMake/CMake-3.6.1-use-gnu11.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+diff -ur cmake-3.6.1.orig/Modules/Compiler/Intel-C.cmake cmake-3.6.1/Modules/Compiler/Intel-C.cmake
```
Comment (boegel) [inline easybuild/easyconfigs/c/CMake/CMake-3.6.1-use-gnu11.patch]: @wpoely86 patch description + author missing, please include any relevant error messages to help Google ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2028.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/dda61edb9778af46b0e6a6724e2a15cf for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f9294a256f50524d7da618da6a47415b for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
build2901.metapod.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU           X5675  @ 3.07GHz, Python 2.7.5
See https://gist.github.com/36abfbd9f5b35ff8e1ec57b35b406994 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2001.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d3e298cc6360bb481fd8606980d3ba36 for a full test report.

**Pull request merged**